src/components/parameters/ValueSourceBadge.svelte

```diff
@@ -29,7 +29,7 @@
     'value-source-badge-dot--preset': source === 'preset',
     'value-source-badge-dot--user': source === 'user on model' || source === 'user on preset',
   });
-  $: {
+  $: if (typeof window !== 'undefined') {
```

Comment:
Is a better check here to see if we're running in the browser?

https://svelte.dev/docs/kit/$app-environment#browser

Reply:
yup thanks, forgot about that 🤦